rpcs3/Emu/RSX/VK/VKTextOut.h

```diff
@@ -255,6 +255,11 @@ namespace vk
 
 		void init(vk::render_device &dev, VkRenderPass &render_pass)
 		{
+			if (render_pass == 0)
```

Comment:
We use BGRA8_UNORM for backbuffer surfaces, and the text-out pipe writes directly to it. If you intend to keep this as a debug assertion for documentation reasons just simplify this to a one line `verify(HERE) renderpass != VK_NULL_HANDLE;` since its technically impossible to hit this assert.